packages/sites/src/get-domain.ts

```diff
@@ -15,7 +15,7 @@ export function getDomains(
   requestOptions?: IRequestOptions
 ): Promise<any> {
   const apiUrl = getHubApiUrl(requestOptions);
-  const url = `${apiUrl}/utilities/domains`;
+  const url = `${apiUrl}/api/v3/domains`;
```

Comment:
why not use `_getDomainServiceUrl()` here/

Reply:
good point i'll refactor that

Reply:
looks like a type mismatch--`_getDomainServiceUrl` expects `IHubRequestOptions` which we don't have in `getDomains`

Reply:
you can change that to accept `hubRequestOptions: IHubRequestOptions | hubApiUrl: string`